ledger-tool/src/main.rs

```diff
@@ -847,6 +848,16 @@ fn main() {
                 .hidden(hidden_unless_forced())
                 .help(BlockVerificationMethod::cli_message()),
         )
+        .arg(
+            Arg::with_name("unified_scheduler_handler_threads")
+                .long("unified-scheduler-handler-threads")
+                .value_name("COUNT")
+                .takes_value(true)
+                .validator(|s| is_within_range(s, 1..))
```

Comment:
i thought about introducing some sensible upper bound like 1024, but opted not to do so because this flag doesn't need to be friendly that much and that restriction would hamper some pathological stress testing (again, this isn't strong point, as this is veerry niche use case).